autogen.des

```diff
@@ -23,7 +24,11 @@ else
 fi
 
 rm -f configure
-(cd $SRCDIR && . ./autogen.sh 2>&1 | egrep -v "(subdir-objects|is in a subdirectory)")
+TREEDIR=`pwd`
```

Comment:
You are removing the output filter here. And I think a subshell is not that bad after all, just the return code needed to be saved.